ietf-te-types.yang

```diff
@@ -503,18 +503,34 @@ module ietf-te-types {
         "ITU-T G.873.1, G.8031, G.8131";
   }
 
-  identity lockout-of-protection {
+  identity local-lockout-of-protection {
     base lsp-protection-state;
     description
-        "A Loss of Protection (LoP) command is active.";
+        "A local Loss of Protection (LoP) command is active.";
     reference
         "ITU-T G.808, RFC 4427";
   }
 
-  identity forced-switch {
+  identity remote-lockout-of-protection {
     base lsp-protection-state;
     description
-        "A forced switch (FS) command is active.";
+        "A remote Loss of Protection (LoP) command is active.";
+    reference
+        "ITU-T G.808, RFC 4427";
+  }
+	
+  identity local-forced-switch {
```

Comment:
suggest splitting into 2:
- action
- at which node the action was applied

Reply:
node may be identified by node-id?
The clear command can be only be allowed if applied on the same node-id

Reply:
The scenario we need to understand is the one in which the command e.g. lockout-of-protection is applied to a node that is outside the control of the server e.g. a node controller or a PNC. That means that this server cannot remove the operator command when it receives a   "clear" action from its client. So the problem is not related to control the node to which the server should apply an operator command but to report whether the operator command has been applied to a node controlled by the server (e.g. PNC)  or not.

Reply:
action
node protection-group-ingress or/both
node protection-group-egress

and in state, 2 leafs:
node protection-group-ingress or/both
node protection-group-egress